src/controllers/CategoryController.ts

```diff
@@ -19,8 +19,10 @@ export class CategoryController {
       res.json(Categories)
       return Categories
     } catch (error: any) {
-      res.status(500).json({ error: error.message })
-      throw new Error(`Error retrieving categories`)
+      res
+        .status(500)
+        .json({ error: 'Internal server error, please try again alter' })
```

Comment:
This is a very general error message - Try to be more specific and paste the error itself in addition to your customized message!